oh_staff_ui/views_utils.py

```diff
@@ -93,6 +96,8 @@ def get_edit_item_context(item_id: int) -> dict:
         "publisher_formset": publisher_formset,
         "resource_formset": resource_formset,
         "subject_formset": subject_formset,
+        "relatives": relatives,
+        "top_parent": top_parent,
```

Comment:
Since `relatives` contains `top_parent`, it seems redundant to include `top_parent` explicitly in the context.